src/Evolution/Systems/CurvedScalarWave/CalculateGrVars.hpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/Initialization/InitialData.hpp"
+#include "Evolution/Initialization/Tags.hpp"
+#include "Evolution/Systems/CurvedScalarWave/System.hpp"
+#include "PointwiseFunctions/AnalyticData/Tags.hpp"
+
+namespace CurvedScalarWave::Actions {
+
+/// \ingroup ActionsGroup
+/// \brief Action that initializes or updates items related to the
+/// spacetime background of the CurvedScalarWave system
+///
+/// DataBox changes:
+/// - Adds:
+///   * `CurvedScalarWave::System::spacetime_tag_list`
+/// - Removes: nothing
+/// - Modifies: nothing
+
+template <typename System>
```

Comment:
Can this action be turned into a `Mutator` and used with the `MutateApply` action? That might simplify it and the test quite a bit. If it's annoying, don't worry about it

Reply:
I think it would complicate things because you want to alter so many tags at once. I would assume it would boil down to writing out the signature and body for each tag individually or some messy combination of variadic templates and tag lists so this just seems easier.

Reply:
Okay, sounds good! That's what I was worried would be the case here. Thanks!